pkg/shell/topnav.jsx

```diff
@@ -86,6 +88,12 @@ export class TopNav extends React.Component {
         return null;
     }
 
+    handleModeClick = (isSelected, event) => {
+        const theme = event.currentTarget.id;
+        this.setState({ theme: theme });
+        localStorage.setItem('shell:mode', theme);
+    }
```

Comment:
These 5 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17914-20221115-124418-c7be4864-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)